### PR TITLE
Spec sheet update proposal

### PR DESCRIPTION
**Pull request by owalk**

### CODE REVIEW

**Comment by owalk** [comment]
These are all just ideas and can be changed if someone thinks of something better. I'm adding everyone as a reviewer. Please leave comments if anything could be improved

File: docs/specSheet.md [hunk 1]
```diff
@@ -35,6 +41,16 @@ Lunk negative interactions:
...
 * ingest non-tasty things
 
+## Game Flow
+The game will start with Lunk spawning into a level facing towards the center of the map. Lunk will be always be running.
+
+There will be a set amount of enemies on the map.
+There will be a set amount of damsels in distress on the map.
+The map will have walls and trees that act as barriers.
+
+The player will be insentivised to plan an optimal route to save the maximum amount of pretty ladies. Levels should be structured such that deviating from the optimal route should have negative consequences, but not be unfun to play.
```
Comment (Sean-Nishi) [inline docs/specSheet.md:51]: incentivized

File: docs/specSheet.md [hunk 2]
```diff
@@ -35,6 +41,16 @@ Lunk negative interactions:
 * crush damsel
 * ingest non-tasty things
 
+## Game Flow
+The game will start with Lunk spawning into a level facing towards the center of the map. Lunk will be always be running.
+
+There will be a set amount of enemies on the map.
+There will be a set amount of damsels in distress on the map.
+The map will have walls and trees that act as barriers.
```
Comment (Sean-Nishi) [inline docs/specSheet.md:49]: We could add boulders as another barrier in the future. Maybe keep it vague like "map will have various obstacles that act as barriers."?

Comment (owalk) [reply docs/specSheet.md]: Changed from walls and trees to "land scape entities"

File: docs/specSheet.md (hunk 1 above)
Comment (owalk) [reply docs/specSheet.md]: fixed

File: docs/specSheet.md [hunk 3]
```diff
@@ -93,6 +109,23 @@ The player status is determined by the current player rotation. The options are
 
 The player status determines which animation frame will be active.
 
+## Map
+There are multiple map files stored per level. a map file is a CSV file containing the location of a tilesheet asset. Each layer of objects within the map will have it's own CSV file.
```
Comment (Sean-Nishi) [inline docs/specSheet.md:113]: Capitalize beginning of sentence

**Review by Sean-Nishi — DISMISSED** [review]
Looks good. Just one spelling error

**Review by Sean-Nishi — APPROVED** [review]

**Review by Sean-Nishi — APPROVED** [review]

**Review by Sean-Nishi — APPROVED** [review]
Maybe change line 41 to end with "Lunk will always be moving". Unless we want different player speeds. For running vs walking

**Review by Sean-Nishi — APPROVED** [review]

**Comment by owalk** [comment]
> Maybe change line 41 to end with "Lunk will always be moving". Unless we want different player speeds. For running vs walking

This has been changed

**Pull request merged**